lib/routes/shopify/apps/search.ts

```diff
@@ -0,0 +1,76 @@
+import { Data, DataItem, Route } from '@/types';
+import type { Context } from 'hono';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { baseURL } from './const';
+
+export const route: Route = {
+    path: '/apps/search/:q',
+    example: '/shopify/apps/search/flow',
+    parameters: { q: '需要搜索的 App' },
+    name: 'Shopify App store search',
+    maintainers: ['PrintNow'],
+    handler,
+};
+
+async function handler(ctx?: Context): Promise<Data> {
+    if (!ctx) {
+        throw new Error('未传入 ctx');
+    }
+
+    const { q = '' } = ctx.req.param();
+    const response = got.get(`${baseURL}/search`, {
+        searchParams: {
+            q,
+        },
+        responseType: 'text',
```

Comment:
This is the [default value](https://github.com/sindresorhus/got/tree/v11.8.6?tab=readme-ov-file#responsetype), no need to specify again.